lib/order.rb

```diff
@@ -0,0 +1,25 @@
+module Order
+
+  def select_dish(name,quantity)
+    raise "Sorry, dish is not available" unless dish_available?(name)
+    @order[name] = quantity
+    @order
+  end
+
+  def submit_order(amount)
+    raise "Please key in the right amount" unless right_amount?(amount)
+    "Your order has been placed successfully. You will receive a confirmation text message."
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)